packages/jwt-service/__tests__/restAgent.test.ts

```diff
@@ -29,7 +29,7 @@ const port = 13213
 const basePath = '/agent'
 
 const DID_METHOD = 'did:jwk'
-// const PRIVATE_KEY_HEX = '7dd923e40f4615ac496119f7e793cc2899e99b64b88ca8603db986700089532b'
+119f7e793cc2899e99b64b88ca8603db986700089532b'
```

Comment:
Whats up here?

Reply:
Fixed already.
Bram requested to remove these, somehow IJ must have messed up my selection